### PR TITLE
Pixelate range thumb and tracker

### PR DESCRIPTION
**Pull request by telmen**
It seems like everything is pixelated except the range track and thumb. And I decided to give it a try and implemented this by applying background image.  

Hope you like it.

### CODE REVIEW

**Comment by SaraVieira** [comment]
Hey!

That would be awesome but there seems to be something not going very well in the deployed URL: https://600eefff7bfc6c000798c322--nifty-hypatia-629e6f.netlify.app/ :/

**Comment by telmen** [comment]
Hey, which browser are you using?

**Comment by SaraVieira** [comment]
I am using the last version of chrome

CleanShot 2021-01-27 at 13 05 23

**Comment by telmen** [comment]
Now it will serve respective assets locally. Please check. :)

File: pages/index.js [hunk 1]
```diff
@@ -44,7 +44,7 @@ export default function IndexPage() {
         </div>
       </div>
       <footer className="flex items-center justify-center mb-6">
-        <a href="https://github.com/SaraVieira/pixel-splash">
+        <a href="https://github.com/SaraVieira/pixler">
```
Comment (SaraVieira) [inline pages/index.js:47]: Thank you!

**Review by SaraVieira — APPROVED** [review]
AAAA
IT LOOKS AMAZING!


Thank you :D

**Pull request merged**